### PR TITLE
Minor Item Additions To Synth Vendor

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Adds several items to synth vendors. Drip vendor can vend more standard issue uniforms, a sechud and all colour varients of a marine helmet. Equip vendor can vend a fuel cannister and all colour varients of the synth utility vest 

# Explain why it's good for the game

Brings equip up to speed with CTs, adds more cosmetic options. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Synthetic equipment vendor now can vend fuel cannisters and all colour variants of the synthetic utility vest.
add: Synthetic cosmetic vendor now can vend all colour variants of the standard Marine helmet, MP and Combat Technician uniforms, the welder chestrig and security hud glasses.
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/crew/synthetic.dm [hunk 1]
```diff
@@ -28,6 +29,7 @@
 		list("Toxin Firstaid Kit", 6, /obj/item/storage/firstaid/toxin, null, VENDOR_ITEM_REGULAR),
 		list("Oxygen Firstaid Kit", 6, /obj/item/storage/firstaid/o2, null, VENDOR_ITEM_REGULAR),
 		list("Radiation Firstaid Kit", 6, /obj/item/storage/firstaid/rad, null, VENDOR_ITEM_REGULAR),
+		list("Empty Adv Firstaid Kit", 1, /obj/item/storage/firstaid/adv/empty, null, VENDOR_ITEM_REGULAR),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/crew/synthetic.dm:32]: I don't really like that we've started handing these out empty. Let's take this out.

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by Steelpoint** [comment]
Update, new items added

- All colour variants of the Synthetic armour can be vended in the equipment vendor.
- All colour variants of the standard Marine helmet can be bended in the cosmetic vendor
- The welder chest rig can be vended in the cosmetic vendor
- Did some tidy up of the categories of the cosmetic vendor

File: code/game/machinery/vending/vendor_types/crew/synthetic.dm [hunk 2]
```diff
@@ -215,7 +225,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_synth_snowflake, list(
 	list("Shoes, Yellow", 12, /obj/item/clothing/shoes/yellow, null, VENDOR_ITEM_REGULAR),
 	list("Shoes, Seegson", 24, /obj/item/clothing/shoes/dress, null, VENDOR_ITEM_REGULAR),
 
-	list("HELMET", 0, null, null, null),
+	list("HAT", 0, null, null, null),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/crew/synthetic.dm:228]: Pedantic but "HEADWARE" is likely better

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Otherwise looks fine oh they changed the little buttons on this prompt they're fatter neato

**Comment by Steelpoint** [comment]
requested changes made

File: code/game/machinery/vending/vendor_types/crew/synthetic.dm (hunk 2 above)
Comment (morrowwolf) [reply code/game/machinery/vending/vendor_types/crew/synthetic.dm]: TFW

**Pull request merged**